README.md

```diff
@@ -1 +1,232 @@
-# kernel-module-imx-pwm-led
+# Freescale i.MX PWM LED Linux Kernel Module
```

Comment:
Is this standard? normally copyright and license are always at the end below Overview?

Reply:
No it's not standard, agreed I'll move it

Reply:
Fixed

Reply:
Epic. sorry I can't really review any of the other stuff.  :)